---
layout: default
---

> National Science Foundation Postdoctoral Fellow  
> Dunn Laboratory, Yale University 

* TOC
{:toc}
# Research

I am an evolutionary biologist; I study the relationships between organisms to learn how biodiversity is generated. My research uses field work, lab work, and computational biology to investigate evolution across scales of life.

Big questions in my work are:

- How much variation do we expect to observe across evolutionary time? What is our null hypothesis?
- To what extent does development constrain or augment the evolutionary process?
- How are species related, and what do phylogenies reveal about patterns in trait evolution?

My work centers on the curation, comparison, and visualization of large trait datsets. To do this, I write new software that facilitates gathering and comparing high-dimensional datasets across species.

### Evolutionary inference for big developmental data

In my current research, I am developing tools for evolutionary analysis of gene expression data that emerge from single-cell sequencing and other RNA-seq technologies. This work combines mathematics, bioinformatics, and developmental and evolutionary biology to build a new framework for analyzing these data.

In this project and throughout my research, I advocate for building robust [null hypotheses](https://doi.org/10.1242/dev.178004) that describe how much variation we expect to observe when we compare across species. Check out this twitter thread for a brief explainer:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://twitter.com/redmakeda?ref_src=twsrc%5Etfw">@redmakeda</a> and I have a new paper talking about null hypotheses in evo-devo. A null hypothesis answers the question:<br><br>“How much difference do we expect when we study development in different species?”<a href="https://t.co/DdZKDEYUfM">https://t.co/DdZKDEYUfM</a></p>&mdash; Sam Church (@shchurch) <a href="https://twitter.com/shchurch/status/1254861903882530816?ref_src=twsrc%5Etfw">April 27, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>    

### Life-history evolution and insect eggs

I study how developmental biology can modulate trait evolution. Life-history theory describes how traits like size, shape, and reproductive features are related across organsims and over evolutionary time.

In my PhD work, I curated a large dataset of insect reproductive traits like egg size and shape by digitizing records from the published literature. Our team used this dataset to ask questions like:
*   Which insects lay the biggest and smallest eggs?
*   How is the size of an egg related to the development of the embryo inside?
*   What can the vast range of egg shapes and sizes tell us about the evolutionary process?

![picticornis_egg](/assets/img/picticornis_egg.png)
Egg of the Hawaiian fly *Drosophila picticornis*

Our work was recently [published](https://doi.org/10.1038/s41586-019-1302-4) in the journal _Nature_. For a deep dive on insect egg shape and size, check out our custom [data visualization](/dataviz/index.html)

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Insects lay eggs of all shapes and sizes! This makes them a great system for exploring the way shape and size evolve. <br><br>This non-technical thread highlights the findings of our new paper in <a href="https://twitter.com/Nature?ref_src=twsrc%5Etfw">@nature</a> by <a href="https://twitter.com/shchurch?ref_src=twsrc%5Etfw">@shchurch</a> <a href="https://twitter.com/brunoasm?ref_src=twsrc%5Etfw">@brunoasm</a> <a href="https://twitter.com/redmakeda?ref_src=twsrc%5Etfw">@redmakeda</a> and me.<a href="https://t.co/cRPG8YRHbc">https://t.co/cRPG8YRHbc</a> <a href="https://t.co/e717I8wzbS">pic.twitter.com/e717I8wzbS</a></p>&mdash; Seth Donoughe (@seth_donoughe) <a href="https://twitter.com/seth_donoughe/status/1146469955380137984?ref_src=twsrc%5Etfw">July 3, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

I also study the organ that makes insect eggs: the insect ovary. Our recent [publication in _Proceedings of the Royal Society B_](https://doi.org/10.1098/rspb.2021.0150) found that understanding ovary development is critical to interpreting patterns in the ovary variation across species. 

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We have a new publication at <a href="https://twitter.com/RSocPublishing?ref_src=twsrc%5Etfw">@RSocPublishing</a> <a href="https://twitter.com/hashtag/ProcB?src=hash&amp;ref_src=twsrc%5Etfw">#ProcB</a> on offspring size, number, and the evolution of the insect ovary! In it we test for general patterns in the shape and structure of ovaries across insects. Here’s a few things we learned from that work: <a href="https://t.co/XgZ9eXVirW">https://t.co/XgZ9eXVirW</a></p>&mdash; Sam Church (@shchurch) <a href="https://twitter.com/shchurch/status/1390011348315217922?ref_src=twsrc%5Etfw">May 5, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
  
### Hawaiian _Drosophila_: model clade for evo-devo

Millions of years ago, _Drosophila_ fruit flies found their way to the Hawaiian islands. Now there are many hundreds of species found on Hawaii and nowhere else in the world.

![picticornis](/assets/img/picticornis.png)
Adult Hawaiian fly *Drosophila picticornis*

Given their close relationship to model species like the fruit fly _Drosophila melanogaster_, this group is primed to become a model clade for evo-devo. In my research, I combine next-gen sequencing and comparative methods to uncover the [evolutionary history of these species](https://doi.org/10.1093/molbev/msac012).

Specifically, I investigate variation in developmental mechanisms that underlies biodiversity in this group. In my PhD work, I sequenced gene expression data across tissues from a dozen Hawaiian species. This work, recently [released as a preprint](https://doi.org/10.1101/2021.11.30.470652), demonstrates how comparative methods can be used to detect evolutionary changes in gene expression across organs and species.

### Phylogenetic software and methods

Phylogenies are hypotheses about the relationships between organisms. Working with Casey Dunn and Joe Ryan, we wrote software to statistically compare phylogenies.

![sowhat](/assets/img/sowhat.png)

We call our software `sowhat`, feel free to [check it out](https://github.com/josephryan/sowhat), and reach out if you need any help using it.

This software has been deployed by many researchers, including myself in work on the evolutionary history of invertebrate lineages (e.g. [Cnidaria](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0139068), [Siphonophora](https://www.sciencedirect.com/science/article/pii/S1055790318300460)).

# Free Adult Education

I am a passionate educator, and have substantial experience both in the university classroom, and as a volunteer educator to adults in my community.

![english_class](/assets/img/english_class.png)

I previously ran the Boston-Cambridge English program for adults. These classes primarily serve Portuguese and Spanish speaking adults at all English proficiency levels.

[Boston-Cambridge English Class](https://www.facebook.com/groups/bostoncambridgeenglishclass)

# Publications

Check out my [Google Scholar profile](https://scholar.google.com/citations?user=4fMce0EAAAAJ&hl=en) for a list of all publications.

# Contact

email: samuelhchurch@gmail.com



